extensions/sql-database-projects/src/models/project.ts

```diff
@@ -139,9 +145,7 @@ export class Project implements ISqlProject {
 		const proj = new Project(projectFilePath);
 
 		proj.sqlProjService = await utils.getSqlProjectsService();
-
 		await proj.readProjFile();
-		await proj.updateProjectForRoundTrip();
```

Comment:
I don't see this getting called somewhere else after this was removed here. Am I missing where it is now?

Reply:
Uhh, that's over in the `projectController.ts`, in the code that prompts the user if they want to update... that I meant to include in this PR.

Reply:
Okay, this is added now, and I unskipped the test that checked it.